### DISCUSSION TITLE
PDF support anytime soon?

### DISCUSSION THREAD

**Discussion by murkomarko** [brianpetro/obsidian-smart-connections#700] [Q&A]
I see that it's in the roadmap to support pdf files. Is there any idea when it's coming?

**Comment by hittjwX** [comment]
Adobe Acrobat has a very powerful AI Assistant for PDFs. If you are the author, you can have the text in Obsidian. Having another author's work in your vault may distract from original ideas.

**Comment by coulas** [comment]
My notes is about building my understanding and knowledge sitting on shoulders of giant that came before me.
I buy eBooks to read them, note them, zettelkasten them.
I little help form my friend would be welcomed.
My knowledge never come from nowhere, and I want to be able to state my sources in all my work.
In short, I am looking forward to local embeddings on pdf :-)

**Reply by hittjwX** [reply]
Yes, Adobe Acrobat will help you do that. The difference is that you would read them outside of Obsidian. If you want to use AI to help with understanding, you can do that in Adobe Acrobat. From a source reference standpoint, you can use Zotero.

All these PDFs can still be in your Obsidian vault. However, I'm sharing my experience here. I have a terabyte of PDFs on a network drive. Every document has been enhanced with OCR, searchable from the operating system, and some have the text extracted. That share is called `Library`.

When I want to explore new ideas, I go to the library like I did 20 years ago. I start with a purpose and a few questions, then take notes in Obsidian.

If the AI in Smart Connections reads those PDFs, it doesn't significantly improve my understanding of those materials. However, reading the PDFs still allows me to cite my sources.

The AI approach may incorporate materials that aren't accurate or relevant to my purpose. Seeing the original context has more meta-value in my research and writing.

Just something to think about.
